api/nodes/serializers.py

```diff
@@ -124,8 +129,22 @@ def update(self, instance, validated_data):
         the request to be in the serializer context.
         """
         assert isinstance(instance, Node), 'instance must be a Node'
+        auth = self.get_user_auth(self.context['request'])
         for attr, value in validated_data.items():
-            setattr(instance, attr, value)
+            if attr == 'tags':
+                old_tags = set([tag._id for tag in instance.tags])
+                if value:
+                    current_tags = set(literal_eval(value))
```

Comment:
`DateTimeFields` will attempt to parse out the input value and deserialize to a `datetime`

https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/fields.py#L1003

It's possible that your input format is incorrect.